test/cypress-e2e/cypress/support/pages/2-group-application-pages/02-organization-info-group-application-CCFRI.js

```diff
@@ -0,0 +1,88 @@
+function handleCardWithin(card, data) {
+    cy.wrap(card).within(() => {
+        Object.entries(data).forEach(([key, value]) => {
+            cy.getByLabel(`${key}`).invoke('val', value).trigger('input')
+        });
+  });
+}
+
+class CcfriApplication{
+    startCcfriApplication() {
+        let optInOrOut
+        let parentFees
+        let closures
+        
+        cy.fixture('groupApplicationDataCCFRI').then((data)=> {
+            optInOrOut = data.optInOrOut
+            parentFees = data.parentFees
+            closures = data.closures
+        })
+
+        cy.then(()=> {
+            // Update any test data here --> NOTE: Some data may need to be updated in related fixture file directly
+            optInOrOut = optInOrOut.optIn
+            let parentFeeCategories = parentFees.parentFeeCategories
+            let paymentFrequency = parentFees.frequency.monthly
+            let closureCharges = closures.closureCharges.chargeForClosures
+            let startDate = closures.startDate
+            let endDate = closures.endDate
+            let closureReason = closures.closureReason
+            let fullFacilityClosureStatus = closures.fullFacilityClosureStatus.notFullFacilityClosure
```

Comment:
I will let you check these all but they can all likely be const. If a variable is not re-assigned then const is a better choice than let. Please check other files as well.